Moodlet/app/src/main/java/com/cmput3owo1/moodlet/models/EmotionalState.java

```diff
@@ -47,16 +43,9 @@
      */
     @Override
     public String toString(){
-        return this.display_name;
+        return this.displayName;
     }
 
-    /**
-     * Returns the display name of the emotional state when called.
-     * @return The display name
-     */
-    public String getDisplayName(){
-        return this.display_name;
-    }
```

Comment:
I think we should leave this in and change the `toString` method instead

Reply:
the "toString" method is needed because the dropdown/spinner uses it internally to get the options to display. I removed this method so we don't have two methods that do the same thing

Reply:
I still feel that we should keep this function in because, even though they are returning the same thing, their intended functionality is essentially different. If I were using the `EmotionalState` class on its own (not with the spinner), it doesn't seem as intuitive to me that I would have to use the `toString` method to get only the display name rather than an explicit getter method.

Reply:
Okay, that makes sense. I'll change it back